STATUS.md

```diff
@@ -279,15 +279,48 @@ This is partially defined in `UDL` and `code`.
+
+### ZcashSaplingNote
+
+* Original type: [zcash_primitives::sapling::Note](https://docs.rs/zcash_primitives/0.10.2/zcash_primitives/sapling/note/struct.Note.html)
+
+| Object/Method name                 |    Score        |        UDL         |        Code        |       Tests        | Docs  |
+| ---------------------------------- | :----------:    | :----------------: | :----------------: | :----------------: | :---: |
+| ZcashSaplingNote::from_parts()  | :red_circle:  | :white_check_mark: | :white_check_mark:  |                    |  :white_check_mark:     |
+| ZcashSaplingNote::recipient()   |                  |                    |                    |                    |       |
+| ZcashSaplingNote::value()       |                  |                    |                    |                    |       |
+| ZcashSaplingNote::rseed()       |                  |                    |                    |                    |       |
+| ZcashSaplingNote::nf()          |                  |                    |                    |                    |       |
+| ZcashSaplingNote::cmu()          |                  |                    |                    |                    |       |
+| ZcashSaplingNote::rcm()          |                  |                    |                    |                    |       |
+| ZcashSaplingNote::generate_or_derive_esk()  |                  |                    |                    |          |       |
+
+
+### ZcashSaplingNoteValue
+
+* Original type: [zcash_primitives::sapling::value::NoteValue](https://docs.rs/zcash_primitives/0.10.2/zcash_primitives/sapling/value/struct.NoteValue.html)
+
+| Object/Method name                 |    Score        |        UDL         |        Code        |       Tests        | Docs  |
+| ---------------------------------- | :----------:    | :----------------: | :----------------: | :----------------: | :---: |
+| ZcashSaplingNoteValue::inner()  | :red_circle:  | :white_check_mark: | :white_check_mark:  |                    |  :white_check_mark:     |
+| ZcashSaplingNoteValue::from_raw()  | :red_circle:  | :white_check_mark: | :white_check_mark:  |                    |  :white_check_mark:     |
+
+
+### ZcashSaplingDiversifiedTransmissionKey
+
+* Original type: [zcash_primitives::sapling::keys::DiversifiedTransmissionKey](https://docs.rs/zcash_primitives/0.10.2/zcash_primitives/sapling/keys/struct.DiversifiedTransmissionKey.html)
+
+| Object/Method name                 |    Score        |        UDL         |        Code        |       Tests        | Docs  |
+| ---------------------------------- | :----------:    | :----------------: | :----------------: | :----------------: | :---: |
+| ZcashSaplingDiversifiedTransmissionKey::from_parts()  | :red_circle:  | :white_check_mark: | :white_check_mark:  |                    |  :white_check_mark:     |
 
 ### ZcashOrchardSpendingKey
```

Comment:
Basically updating the `ZcashPaymentAddress` and adding other new objects used by its methods.